src/applications/static-pages/view-modify-dependent/686-cta/containers/Form686CTA.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import { connect } from 'react-redux';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+const Form686CTA = props => {
+  let content;
+  if (props.includedInFlipper === undefined) {
+    content = <LoadingIndicator message="Loading..." />;
+  } else if (props.includedInFlipper === true) {
```

Comment:
I don't think I'd show a loader, because it would be kind of a fake-out to show a loader but render empty 🙂 I think I'd just do -

```jsx
// At the top
if (!props. includedInFlipper) {
    return null;
}

return (
   // content...
);
```

There might be be a flicker but that's okay. The Flipper API route is usually really fast so it shouldn't be much.

Reply:
@ncksllvn Yeah I didn't know if the flicker was gonna pass 508 so I went with the loader but I think you are right here. Updated.